tests/TypeTests.hs

```diff
@@ -7,7 +7,38 @@ import Test.Tasty (TestTree)
 import Test.Tasty.Hspec (testSpec, describe, it, shouldBe)
 
 import GraphQL.Definitions
+  ( GraphQLEnum(..)
+  , Enum
+  , Object
+  , Field
+  , Argument
```

Comment:
these long import lists are kinda uncool but I don't know what to do instead. I dislike implicit all-imports as well. Qualified imports make the tests less readable. (not actionable)